packages/recommend-react/package.json

```
@@ -30,7 +30,8 @@
   },
   "dependencies": {
     "@algolia/recommend-core": "1.0.0",
-    "@algolia/recommend-vdom": "1.0.0"
+    "@algolia/recommend-vdom": "1.0.0",
+    "dequal": "2.0.2"
```

We don't need to have a hard dependency here, it increases the likelihood of users getting duplicated versions of `dequal` in their projects.

```suggestion
    "dequal": "^2.0.0"
```